frontend/app/controllers/skill-search.js

```diff
@@ -14,51 +13,138 @@ export default class SkillSearchController extends Controller {
   @service
   router;
 
-  @tracked
-  levelValue = this.level || 1;
-
-  @tracked
-  interestValue = this.interest || 1;
-
-  currentSkillId = this.skill_id;
-
-  @tracked
-  skills = this.store.findAll("skill", { reload: true });
+  @action
+  initFilters() {
+    let filters = [];
+    let skill_id = new URLSearchParams(window.location.search).get("skill_id");
```

Comment:
Is the `URLSearchParams` default? I don't see it anywhere else in the code so maybe there is a cleaner way.